wiki/Client/Release_stream/Lazer/Feature_comparison/en.md

```diff
@@ -0,0 +1,438 @@
+# osu!(lazer) feature comparison
+
+*For a more condensed list of features, see [Help centre/Upgrading to lazer § Feature comparison](/wiki/Help_centre/Upgrading_to_lazer#feature-comparison)*
+
+This page documents all new and missing features compared to osu!(stable), as well as differences affecting gameplay.
+
+## New features
+
+- Cross-platform support
+  - Native Windows 8.1+, macOS, Linux, iOS, and Android support, 
+  - Mobile input methods for all game modes
+- Input
+  - Built-in tablet driver
+  - Midi controller support
+  - Gamepad support
+- Redesigned interface
+  - First-run setup wizard
+  - Storyboards in the main menu
+  - Rewind F2
+  - Hiding difficulties
+  - Soft deletion
+  - More screen scaling options
+  - Settings can be changed everywhere and they'll take effect immediately
+  - New statistics in the results and song select screens
+- Website integration for leaderboards, news, wiki, changelog, beatmap listing, beatmap info, and profile pages
+- Lots of new mods
+- Mod customisation options
+- New multiplayer features
+  - New "playlist" feature
+  - Unlimited lobby sizes
+  - Automatic beatmap queues
+- New editor features
+  - Per-segment curve types for sliders
+  - Slider splitting and merging
+  - New "verify" tab
+- Support for custom game modes
+- Gameplay
+  - Two new built-in skins
+  - Replay seeking
+  - Beatmap (local) offset auto-calibration
+- Built-in skin editor with customisable skinning components 
+  - Song density graph
+  - Performance points counter
+  - Unstable rate counter
+  - Longest combo counter
+  - Custom text components, beatmap information displays
+  - Custom static sprites
+  - Judgement counter
+  - Clicks per second counter
+- Snaking out sliders in the osu! mode
+- No black bar behind the playfield in osu!taiko
+- Timing-based note colouring in osu!mania
+
+<!-- - osu!catch ?-->
+
+## Gameplay differences 
+
+<!--
+- Multiplayer
+  - TAG co-op mode doesn't exist
+-->
+
+### General
+
+#### Song playback ramps up and down when pausing and unpausing
+
+This makes it harder to continue from a pause in the middle of active gameplay, preventing abuse of pausing to some extent.
+
+|  |  |
+| :-- | :-: |
+| Breaks backwards compatibility | No |
+| Classic mod revert support | No |
+| Intentionally changed | Yes |
+| Needs further consideration | Yes |
+
+#### The Flashlight gradient does not match stable
+
+<!-- comparison images for relevant game modes  -->
+
+|  |  |
+| :-- | :-: |
+| Breaks backwards compatibility | No |
+| Classic mod revert support | No |
+| Intentionally changed | No |
```

Comment:
this was recently revisited, at least for osu! mode and should match pretttttttttttty close now

Reply:
how close is close enough i wonder

https://user-images.githubusercontent.com/36758269/219944801-f4aa5916-ff73-4712-9793-77e43b051282.mp4

Reply:
taiko

| stable | lazer |
| :-: | :-: |
| ![screenshot703](https://user-images.githubusercontent.com/36758269/219945185-cad346fa-e063-40cc-a95f-c9532e307894.jpg) | ![osu_2023-02-19_12-31-20](https://user-images.githubusercontent.com/36758269/219945176-9520c2f2-7294-475f-bbef-28e3a8dc2293.png) |

https://user-images.githubusercontent.com/36758269/219945689-6cd4015f-0cce-4749-a034-e11a3f145cd8.mp4

flashlight size seems to match at least

https://user-images.githubusercontent.com/36758269/219945508-56dd8b85-5bcc-4358-9003-bd338de2df7d.mp4

Reply:
the absolute state of lazer mania flashlight
| stable | lazer |
| :-: | :-: |
| ![stable-mania-flashlight](https://user-images.githubusercontent.com/36758269/219947171-3aa514b8-5dcc-449f-8c23-b837ca9f51d6.jpg) | ![lazer-mania-flashlight](https://user-images.githubusercontent.com/36758269/219947173-2ddece7d-0564-4658-8700-9c97677c4885.png) |

Reply:
catch seems to match pretty much perfectly


https://user-images.githubusercontent.com/36758269/219947705-265a1133-5df3-4352-8d20-8a663e67cd42.mp4

Reply:
i've split up this section into taiko and mania in https://github.com/ppy/osu-wiki/pull/8791/commits/e57c1a213e8ac9a9cfc50432331e8c2522856aa6